src/server.js

```diff
@@ -58,6 +58,10 @@ module.exports = function() {
 		log.warn("Server is public and set to use LDAP. Set to private mode if trying to use LDAP authentication.");
 	}
 
+	if (config.prefetchMaxImageSize) { // TODO: Remove in v3.0.0
+		log.warn("prefetchMaxImageSize has been removed as of v2.7.0. Use the new prefetchMaxMediaSize option instead.");
```

Comment:
Just to try to be more consistent (it's useful to search-entire-project for "deprecated", though I know there are 2 cases that don't match the other 10ish lol) , what do you think of the following?

```js
log.warn(`$colors.bold("prefetchMaxImageSize") is ${colors.bold.red("deprecated")} and will be removed in The Lounge v3. Use ${colors.bold("prefetchMaxMediaSize")} option instead.`);
```

Reply:
Done